### PR TITLE
Update green and red colors

### PR DESCRIPTION
**Pull request by ErrorNoInternet**
## Summary
I have updated the green and red colors to match the colors from the Discord branding page

## Checklist
- [x] If code changes were made then they have been tested.
    - [x] I have updated the documentation to reflect the changes.
- [ ] This PR fixes an issue.
- [x] This PR adds something new (e.g. new method or parameters).
- [ ] This PR is a breaking change (e.g. methods or parameters removed/renamed)
- [ ] This PR is **not** a code change (e.g. documentation, README, ...)

### CODE REVIEW

**Comment by Rapptz** [comment]
These are the colours of the role selector. If you want to add these they'd have to be under a separate name like `brand_green` or `brand_red`.

**Comment by ErrorNoInternet** [comment]
Colors have been updated

**Comment by Rapptz** [comment]
The names are swapped.

File: discord/colour.py [hunk 1]
```diff
@@ -171,6 +171,11 @@ def dark_teal(cls: Type[CT]) -> CT:
         """A factory method that returns a :class:`Colour` with a value of ``0x11806a``."""
         return cls(0x11806a)
 
+    @classmethod
+    def brand_green(cls: Type[CT]) -> CT:
+        """A factory method that returns a :class:`Colour` with a value of ``0x57F287``."""
```
Comment (Rapptz) [inline discord/colour.py:176]:
```suggestion
        """A factory method that returns a :class:`Colour` with a value of ``0x57F287``.

        .. versionadded:: 2.0
        """
```

File: discord/colour.py [hunk 2]
```diff
@@ -231,6 +236,11 @@ def dark_orange(cls: Type[CT]) -> CT:
         """A factory method that returns a :class:`Colour` with a value of ``0xa84300``."""
         return cls(0xa84300)
 
+    @classmethod
+    def brand_red(cls: Type[CT]) -> CT:
+        """A factory method that returns a :class:`Colour` with a value of ``0xED4245``."""
```
Comment (Rapptz) [inline discord/colour.py:241]:
```suggestion
        """A factory method that returns a :class:`Colour` with a value of ``0xED4245``.

        .. versionadded:: 2.0
        """
```

**Review by Rapptz — APPROVED** [review]
LGTM thanks.

**Pull request merged**